### ISSUE TITLE
Remove `Add new account` and `Add watch-only account` items from account context menu

### ISSUE THREAD

**Issue by benjthayer** [status-im/status-desktop#10682]
Currently, when right-clicking an account from the wallet area, the user sees the options for `Add new account` and `Add watch-only account`. 

These 2 actions should be removed from the account context menu given:
1) There are not technically contextual to that account
2) The user can access those 2 actions from the `+` icon button at the top of the account list or via the context menu invoked by right-clicking the empty space in the wallet list panel

<img width="799" alt="Screenshot 2023-05-15 at 10 40 37" src="https://github.com/status-im/status-desktop/assets/110033914/2430f330-d4a8-4656-891a-6c7405dde425">

This change mainly impacts the Add/Edit wallet flows in the Wallet v2 file but will be changed in the Desktop Settings file also where relevant.
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=13909-410577&t=phJiBTpvcsgpMZOG-4

**Comment by alaibe** [comment]
@benjthayer what happens in case of right click in between account, currently:


![Image](https://github.com/status-im/status-desktop/assets/491074/a46ba13c-0469-4fe0-ac8a-d5ae33f46523)

**Comment by saledjenic** [comment]
> what happens in case of right click in between account, currently:

@alaibe I would keep it in that case (personally). :)

**Comment by benjthayer** [comment]
> @benjthayer what happens in case of right click in between account, currently:
> 
> ![Image](https://user-images.githubusercontent.com/491074/242798857-a46ba13c-0469-4fe0-ac8a-d5ae33f46523.png)

I think we should keep the context menu when right-clicking between accounts - we teach the user that they can click on any empty space in the account list to get the context menu to add a new account so that technically applies if they right-click between accounts. 

I would remove the `Copy address` menu item however - there isn't much reason to keep that item even in the disabled state as the user is not at that point expecting to see any account specific menu items. 

cc. @John-44

**Comment by saledjenic** [comment]
> I would remove the Copy address 

Of course, I am not sure how it appeared there since it was not the case when that was implemented.

> I think we should keep the context menu when right-clicking between accounts

It's not only between accounts, but at the beginning for new users, there is only a single account and there is a big empty space beneath it, so right click on that space should display that menu as well, from me +1 on this. :)

**Comment by benjthayer** [comment]
> It's not only between accounts, but at the beginning for new users, there is only a single account and there is a big empty space beneath it, so right click on that space should display that menu as well, from me +1 on this. :)

Yeah absolutely @saledjenic - that was my meaning when I said 'we teach the user that they can click on any empty space in the account list to get the context menu to add a new account ' - I was referring to when the user first joins and can right-click on the empty space. 👍

**Comment by Cuteivist** [comment]
> I would remove the Copy address menu item however - there isn't much reason to keep that item even in the disabled state as the user is not at that point expecting to see any account specific menu items.

That will be solved in #10507
